jobs/settings.py

```diff
@@ -1,22 +1,12 @@
 import os
 from datetime import timedelta
-
-
-# Force to cast strings to bool
-def str2bool(value: str, default: bool = False) -> bool:
-    cleaned_value = value.lower().strip()
-    if cleaned_value in ['true', 't', '1', 'yes']:
-        return True
-    if cleaned_value in ['false', 'f', '0', 'no']:
-        return False
-    return default
-
+from distutils.util import strtobool
```

Comment:
instead of this I would recommend you to add django-environ that give you casting out of the box. https://github.com/joke2k/django-environ

Reply:
Ah great. Yes, I agreed. Is my first time using django and I'm still exploring the ecosystem.

Reply:
Thanks @jbagot , I did know about this package.
We use https://pypi.org/project/django-kaio/